app/Notifications/CheckoutAssetNotification.php

```diff
@@ -51,6 +54,10 @@ public function __construct(Asset $asset, $checkedOutTo, User $checkedOutBy, $ac
      */
     public function via()
     {
+        if (Setting::getSettings()->webhook_selected == 'microsoft'){
+
+            return [MicrosoftTeamsChannel::class];
```

Comment:
We don't want to return here because it prevents emails from being sent (since they are no longer added to `$notifyBy[]` below). I think using `$notifyBy[] = MicrosoftTeamsChannel::class;` like in other places would fix it.